### PR TITLE
Fix issue #46 - Keyboard backlight remains off after sleep on Catalina

### PR DESCRIPTION
**Pull request by AR-CADE**

### CODE REVIEW

File: AsusSMC/AsusSMC.cpp [hunk 1]
```diff
@@ -163,6 +163,38 @@ OSDictionary *AsusSMC::getDictByUUID(const char *guid) {
...
+    if (powerStateOrdinal == kIOPMPowerOff) {
+        DBGLOG("atk", "Power off");
+        /**
+        *  Sleep, shutdown or reboot detected
+        */
+        setKBLLevel(kbl_level, false, false);
+    } else {
+        DBGLOG("atk", "Waking up");
+        kbl_level = readKBBacklightFromNVRAM();
+        setKBLLevel(kbl_level);
```
Comment (hieplpvip) [inline AsusSMC/AsusSMC.cpp]: Please change this to `setKBLLevel(kbl_level, false, false);`

File: AsusSMC/AsusSMC.cpp [hunk 2]
```diff
@@ -163,6 +163,38 @@ OSDictionary *AsusSMC::getDictByUUID(const char *guid) {
...
+#define kIOPMPowerOff                       0
+#define kNumberOfStates                     2
+
+static IOPMPowerState powerStates[kNumberOfStates] = {
+   {1, kIOPMPowerOff, kIOPMPowerOff, kIOPMPowerOff, 0, 0, 0, 0, 0, 0, 0, 0},
+   {1, kIOPMPowerOn, kIOPMPowerOn, kIOPMPowerOn, 0, 0, 0, 0, 0, 0, 0, 0}
+};
+
+IOReturn AsusSMC::setPowerState(unsigned long powerStateOrdinal, IOService * whatDevice) {
+
```
Comment (hieplpvip) [inline AsusSMC/AsusSMC.cpp]: Please delete this blank line

**Review by hieplpvip — CHANGES_REQUESTED** [review]
Looks good to me. There are 2 small problems, please fix them and I'll merge this. Thanks very much.

File: AsusSMC/AsusSMC.cpp (hunk 1 above)
Comment (AR-CADE) [reply AsusSMC/AsusSMC.cpp]: done, 

also fix a copy-paste mistake that I made on power off, the level is now set explicitly to 0

File: AsusSMC/AsusSMC.cpp (hunk 2 above)
Comment (AR-CADE) [reply AsusSMC/AsusSMC.cpp]: done

**Pull request merged**